css/style.css

```
@@ -14,7 +15,7 @@ body {
   height: 256;
   position: relative;
   overflow-x: auto;
-  background-image: url('../images/water0.png');
+  background-image: url("../images/water0.png");
```

```suggestion
  background-image: url("../images/water0.png");
  transition: background-image 0.3s;
```
`background-image` property is changed by JS in `initialiseSea()` method. I suspect this can cause flash effect, when it should be seamless. You can add `transition` for `background-image` property with timing value that will work for you (0.3s is only example here, please adjust to what you want/think work the. best).